### DISCUSSION TITLE
No alerts in dashboard since upgrade in 2.3.70

### DISCUSSION THREAD

**Discussion by marcRBD** [Security-Onion-Solutions/securityonion#5536] [Unsupported Versions] [answered]
Hi
i install from iso 2.3.51 

`
Checking Docker status

    Docker -------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]

Checking container statuses

    so-aptcacherng ------------------------------------------------------------------------------------------------------------------------------------------------------------ [ OK ]
    so-cortex ----------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-curator ---------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-dockerregistry --------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-elastalert ------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-elasticsearch ---------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-filebeat --------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-idstools --------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-kibana ----------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-kratos ----------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-nginx ------------------------------------------------------------------------------------------------------------------------------------------------------------------ [ OK ]
    so-sensoroni -------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-soc -------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-soctopus --------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-steno ------------------------------------------------------------------------------------------------------------------------------------------------------------------ [ OK ]
    so-strelka-backend -------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-strelka-coordinator ---------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-strelka-filestream ----------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-strelka-frontend ------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-strelka-gatekeeper ----------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-strelka-manager -------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-suricata --------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-telegraf --------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-thehive ---------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-thehive-es ------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-wazuh ------------------------------------------------------------------------------------------------------------------------------------------------------------------ [ OK ]
    so-zeek ------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]

i install from iso 2.3.51 , i updraded,  now i am in **2.3.70** 
**since  that upgrade,** i have no alerts in the dashboard.

tcpdump show traffic on interface, there is some logs for suricata
it seems filebeat is buggy:

`2021-09-16T11:45:07.857Z        DEBUG   [elasticsearch] elasticsearch/client.go:414     Bulk item insert failed (i=49, status=500): {"type":"illegal_state_exception","reason":"pipeline with id [zeek.common] could not be loaded, caused by [org.elasticsearch.ElasticsearchParseException: processor [community_id] doesn't support one or more provided configuration parameters [field]]"}`
`

`c2021-09-16T07:16:18.047Z        ERROR   [publisher_pipeline_output]     pipeline/output.go:180  failed to publish events: temporary bulk send failure`

i have only some alerts of surricata if i put filebeat in debug like it is explained here:
https://docs.securityonion.net/en/2.3/filebeat.html#diagnostic-logging

there is something broken around fields and filebeat i think.
thanks for your help

**Comment by dougburks** [comment]
Take a look at `/opt/so/conf/elasticsearch/ingest/zeek.common` and `/opt/so/saltstack/default/salt/elasticsearch/files/ingest/zeek.common` and see how they compare to:
https://github.com/Security-Onion-Solutions/securityonion/blob/master/salt/elasticsearch/files/ingest/zeek.common

**Reply by JeremyBBD** [reply]
Hi, member of marcRBD's team here.

The files have not been modified, but I hashed them anyway just to make sure:
```
# sha256sum /opt/so/conf/elasticsearch/ingest/zeek.common
958229fa98316d38a879a619c126e0cd1257acc973044584fa26040bc0fc0abe  /opt/so/conf/elasticsearch/ingest/zeek.common
# sha256sum /opt/so/saltstack/default/salt/elasticsearch/files/ingest/zeek.common
958229fa98316d38a879a619c126e0cd1257acc973044584fa26040bc0fc0abe  /opt/so/saltstack/default/salt/elasticsearch/files/ingest/zeek.common
```

These two files are the exact same ones that the one you linked.

**Reply by dougburks** [reply]
Were there any errors when you upgraded to 2.3.70?

What is the output of `sudo docker images`?

If all else fails, you could always perform a fresh installation of 2.3.70 on a separate machine (perhaps just a VM) and then compare the two systems side-by-side to look for differences.

**Comment by marcRBD** [comment]
thanks for the answer, no error when i read  the screen, but for docker images, there is some old versions ? (2.3.52)

REPOSITORY                                                          TAG       IMAGE ID       CREATED        SIZE
ghcr.io/security-onion-solutions/so-thehive-es                      2.3.70    4beff17c0511   5 weeks ago    1.57GB
tddv-nids-soc:5000/security-onion-solutions/so-thehive-es           2.3.70    4beff17c0511   5 weeks ago    1.57GB
ghcr.io/security-onion-solutions/so-influxdb                        2.3.70    37fed31d61a9   5 weeks ago    183MB
tddv-nids-soc:5000/security-onion-solutions/so-influxdb             2.3.70    37fed31d61a9   5 weeks ago    183MB
tddv-nids-soc:5000/security-onion-solutions/so-idstools             2.3.70    4f33c5d974d1   5 weeks ago    61.6MB
ghcr.io/security-onion-solutions/so-idstools                        2.3.70    4f33c5d974d1   5 weeks ago    61.6MB
ghcr.io/security-onion-solutions/so-telegraf                        2.3.70    311dee6c9cab   5 weeks ago    91.5MB
tddv-nids-soc:5000/security-onion-solutions/so-telegraf             2.3.70    311dee6c9cab   5 weeks ago    91.5MB
ghcr.io/security-onion-solutions/so-strelka-manager                 2.3.70    e99b17d426b5   5 weeks ago    21.6MB
tddv-nids-soc:5000/security-onion-solutions/so-strelka-manager      2.3.70    e99b17d426b5   5 weeks ago    21.6MB
tddv-nids-soc:5000/security-onion-solutions/so-redis                2.3.70    a283e76e62d3   5 weeks ago    32.3MB
ghcr.io/security-onion-solutions/so-redis                           2.3.70    a283e76e62d3   5 weeks ago    32.3MB
ghcr.io/security-onion-solutions/so-soc                             2.3.70    1553e77f29b5   5 weeks ago    85.4MB
tddv-nids-soc:5000/security-onion-solutions/so-soc                  2.3.70    1553e77f29b5   5 weeks ago    85.4MB
ghcr.io/security-onion-solutions/so-kratos                          2.3.70    fe64bbfb697d   7 weeks ago    56.1MB
tddv-nids-soc:5000/security-onion-solutions/so-kratos               2.3.70    fe64bbfb697d   7 weeks ago    56.1MB
ghcr.io/security-onion-solutions/so-thehive-cortex                  2.3.70    d3a38490f690   7 weeks ago    1.76GB
tddv-nids-soc:5000/security-onion-solutions/so-thehive-cortex       2.3.70    d3a38490f690   7 weeks ago    1.76GB
ghcr.io/security-onion-solutions/so-zeek                            2.3.70    b35af5727b77   7 weeks ago    573MB
tddv-nids-soc:5000/security-onion-solutions/so-zeek                 2.3.70    b35af5727b77   7 weeks ago    573MB
ghcr.io/security-onion-solutions/so-nginx                           2.3.70    c8ee17546d79   7 weeks ago    87.4MB
tddv-nids-soc:5000/security-onion-solutions/so-nginx                2.3.70    c8ee17546d79   7 weeks ago    87.4MB
ghcr.io/security-onion-solutions/so-grafana                         2.3.70    4dbc4330d4da   7 weeks ago    498MB
tddv-nids-soc:5000/security-onion-solutions/so-grafana              2.3.70    4dbc4330d4da   7 weeks ago    498MB
ghcr.io/security-onion-solutions/so-fleet-launcher                  2.3.70    6693fd38cefe   7 weeks ago    583MB
tddv-nids-soc:5000/security-onion-solutions/so-fleet-launcher       2.3.70    6693fd38cefe   7 weeks ago    583MB
ghcr.io/security-onion-solutions/so-logstash                        2.3.70    afdc45ce9994   7 weeks ago    1.62GB
tddv-nids-soc:5000/security-onion-solutions/so-logstash             2.3.70    afdc45ce9994   7 weeks ago    1.62GB
ghcr.io/security-onion-solutions/so-curator                         2.3.70    e8ea5ec5b542   7 weeks ago    123MB
tddv-nids-soc:5000/security-onion-solutions/so-curator              2.3.70    e8ea5ec5b542   7 weeks ago    123MB
ghcr.io/security-onion-solutions/so-strelka-frontend                2.3.70    74d2713e295a   7 weeks ago    21.4MB
tddv-nids-soc:5000/security-onion-solutions/so-strelka-frontend     2.3.70    74d2713e295a   7 weeks ago    21.4MB
ghcr.io/security-onion-solutions/so-tcpreplay                       2.3.70    4b2a681f2b58   7 weeks ago    865MB
tddv-nids-soc:5000/security-onion-solutions/so-tcpreplay            2.3.70    4b2a681f2b58   7 weeks ago    865MB
ghcr.io/security-onion-solutions/so-strelka-backend                 2.3.70    3bfda0b24027   7 weeks ago    1.06GB
tddv-nids-soc:5000/security-onion-solutions/so-strelka-backend      2.3.70    3bfda0b24027   7 weeks ago    1.06GB
tddv-nids-soc:5000/security-onion-solutions/so-wazuh                2.3.70    a9e256f9c662   7 weeks ago    884MB
ghcr.io/security-onion-solutions/so-wazuh                           2.3.70    a9e256f9c662   7 weeks ago    884MB
ghcr.io/security-onion-solutions/so-soctopus                        2.3.70    7147dd65d256   7 weeks ago    509MB
tddv-nids-soc:5000/security-onion-solutions/so-soctopus             2.3.70    7147dd65d256   7 weeks ago    509MB
ghcr.io/security-onion-solutions/so-pcaptools                       2.3.70    d1119d4ec1a8   7 weeks ago    120MB
tddv-nids-soc:5000/security-onion-solutions/so-pcaptools            2.3.70    d1119d4ec1a8   7 weeks ago    120MB
ghcr.io/security-onion-solutions/so-kibana                          2.3.70    125fbdd5c717   7 weeks ago    2.2GB
tddv-nids-soc:5000/security-onion-solutions/so-kibana               2.3.70    125fbdd5c717   7 weeks ago    2.2GB
ghcr.io/security-onion-solutions/so-suricata                        2.3.70    d48491d3d8ad   7 weeks ago    766MB
tddv-nids-soc:5000/security-onion-solutions/so-suricata             2.3.70    d48491d3d8ad   7 weeks ago    766MB
tddv-nids-soc:5000/security-onion-solutions/so-acng                 2.3.70    b3d372fa1357   7 weeks ago    81.9MB
ghcr.io/security-onion-solutions/so-acng                            2.3.70    b3d372fa1357   7 weeks ago    81.9MB
ghcr.io/security-onion-solutions/so-mysql                           2.3.70    f0b736914d3c   7 weeks ago    250MB
tddv-nids-soc:5000/security-onion-solutions/so-mysql                2.3.70    f0b736914d3c   7 weeks ago    250MB
ghcr.io/security-onion-solutions/so-strelka-filestream              2.3.70    d1a064f34418   7 weeks ago    20.6MB
tddv-nids-soc:5000/security-onion-solutions/so-strelka-filestream   2.3.70    d1a064f34418   7 weeks ago    20.6MB
ghcr.io/security-onion-solutions/so-steno                           2.3.70    8ff7eed5eb03   7 weeks ago    849MB
tddv-nids-soc:5000/security-onion-solutions/so-steno                2.3.70    8ff7eed5eb03   7 weeks ago    849MB
ghcr.io/security-onion-solutions/so-thehive                         2.3.70    cc9798752eff   7 weeks ago    631MB
tddv-nids-soc:5000/security-onion-solutions/so-thehive              2.3.70    cc9798752eff   7 weeks ago    631MB
ghcr.io/security-onion-solutions/so-playbook                        2.3.70    d1441ed8bd43   7 weeks ago    604MB
tddv-nids-soc:5000/security-onion-solutions/so-playbook             2.3.70    d1441ed8bd43   7 weeks ago    604MB
ghcr.io/security-onion-solutions/so-elasticsearch                   2.3.70    7b6d0a4b011e   7 weeks ago    1.57GB
tddv-nids-soc:5000/security-onion-solutions/so-elasticsearch        2.3.70    7b6d0a4b011e   7 weeks ago    1.57GB
ghcr.io/security-onion-solutions/so-elastalert                      2.3.70    eb5fcb0ba76a   7 weeks ago    434MB
tddv-nids-soc:5000/security-onion-solutions/so-elastalert           2.3.70    eb5fcb0ba76a   7 weeks ago    434MB
ghcr.io/security-onion-solutions/so-fleet                           2.3.70    80cd1618ac13   7 weeks ago    83.1MB
tddv-nids-soc:5000/security-onion-solutions/so-fleet                2.3.70    80cd1618ac13   7 weeks ago    83.1MB
ghcr.io/security-onion-solutions/so-filebeat                        2.3.70    231155ee4a4b   7 weeks ago    628MB
tddv-nids-soc:5000/security-onion-solutions/so-filebeat             2.3.70    231155ee4a4b   7 weeks ago    628MB
**tddv-nids-soc:5000/security-onion-solutions/so-zeek                 2.3.52    f8d7bfcb0855   3 months ago   573MB
ghcr.io/security-onion-solutions/so-zeek                            2.3.52    f8d7bfcb0855   3 months ago   573MB
ghcr.io/security-onion-solutions/so-soc                             2.3.52    0f71842b9d58   3 months ago   85.6MB
tddv-nids-soc:5000/security-onion-solutions/so-soc                  2.3.52    0f71842b9d58   3 months ago   85.6MB
ghcr.io/security-onion-solutions/so-kratos                          2.3.52    e5f41ddfffe7   3 months ago   51.8MB
tddv-nids-soc:5000/security-onion-solutions/so-kratos               2.3.52    e5f41ddfffe7   3 months ago   51.8MB
ghcr.io/security-onion-solutions/so-thehive-cortex                  2.3.52    29406cf35711   3 months ago   1.23GB
tddv-nids-soc:5000/security-onion-solutions/so-thehive-cortex       2.3.52    29406cf35711   3 months ago   1.23GB
ghcr.io/security-onion-solutions/so-strelka-backend                 2.3.52    65fe08704a99   3 months ago   1.04GB
tddv-nids-soc:5000/security-onion-solutions/so-strelka-backend      2.3.52    65fe08704a99   3 months ago   1.04GB
tddv-nids-soc:5000/security-onion-solutions/so-thehive              2.3.52    67daddcb050c   3 months ago   631MB
ghcr.io/security-onion-solutions/so-thehive                         2.3.52    67daddcb050c   3 months ago   631MB
tddv-nids-soc:5000/security-onion-solutions/so-logstash             2.3.52    b0a1a8fce56d   3 months ago   1.63GB
ghcr.io/security-onion-solutions/so-logstash                        2.3.52    b0a1a8fce56d   3 months ago   1.63GB
ghcr.io/security-onion-solutions/so-soctopus                        2.3.52    dbca7e2bcbb1   3 months ago   604MB
tddv-nids-soc:5000/security-onion-solutions/so-soctopus             2.3.52    dbca7e2bcbb1   3 months ago   604MB
ghcr.io/security-onion-solutions/so-strelka-frontend                2.3.52    c7c2a7ad140b   3 months ago   21.7MB
tddv-nids-soc:5000/security-onion-solutions/so-strelka-frontend     2.3.52    c7c2a7ad140b   3 months ago   21.7MB
ghcr.io/security-onion-solutions/so-strelka-manager                 2.3.52    7b8dacc30766   3 months ago   21.5MB
tddv-nids-soc:5000/security-onion-solutions/so-strelka-manager      2.3.52    7b8dacc30766   3 months ago   21.5MB
ghcr.io/security-onion-solutions/so-strelka-filestream              2.3.52    aa532840a9ca   3 months ago   20.6MB
tddv-nids-soc:5000/security-onion-solutions/so-strelka-filestream   2.3.52    aa532840a9ca   3 months ago   20.6MB
ghcr.io/security-onion-solutions/so-steno                           2.3.52    164afc0bda28   3 months ago   849MB
tddv-nids-soc:5000/security-onion-solutions/so-steno                2.3.52    164afc0bda28   3 months ago   849MB
ghcr.io/security-onion-solutions/so-pcaptools                       2.3.52    95d447ff4a2b   3 months ago   120MB
tddv-nids-soc:5000/security-onion-solutions/so-pcaptools            2.3.52    95d447ff4a2b   3 months ago   120MB
ghcr.io/security-onion-solutions/so-influxdb                        2.3.52    520f69f3f578   3 months ago   183MB
tddv-nids-soc:5000/security-onion-solutions/so-influxdb             2.3.52    520f69f3f578   3 months ago   183MB
ghcr.io/security-onion-solutions/so-mysql                           2.3.52    4b4fe5bcad58   3 months ago   250MB
tddv-nids-soc:5000/security-onion-solutions/so-mysql                2.3.52    4b4fe5bcad58   3 months ago   250MB
ghcr.io/security-onion-solutions/so-idstools                        2.3.52    ca5396c943c2   3 months ago   55.9MB
tddv-nids-soc:5000/security-onion-solutions/so-idstools             2.3.52    ca5396c943c2   3 months ago   55.9MB
ghcr.io/security-onion-solutions/so-redis                           2.3.52    6a8af2fe145c   3 months ago   32.3MB
tddv-nids-soc:5000/security-onion-solutions/so-redis                2.3.52    6a8af2fe145c   3 months ago   32.3MB
ghcr.io/security-onion-solutions/so-tcpreplay                       2.3.52    1cd6225acd5d   3 months ago   853MB
tddv-nids-soc:5000/security-onion-solutions/so-tcpreplay            2.3.52    1cd6225acd5d   3 months ago   853MB
ghcr.io/security-onion-solutions/so-fleet-launcher                  2.3.52    67e8738703da   3 months ago   591MB
tddv-nids-soc:5000/security-onion-solutions/so-fleet-launcher       2.3.52    67e8738703da   3 months ago   591MB
ghcr.io/security-onion-solutions/so-grafana                         2.3.52    8d98b23d8c42   3 months ago   499MB
tddv-nids-soc:5000/security-onion-solutions/so-grafana              2.3.52    8d98b23d8c42   3 months ago   499MB
ghcr.io/security-onion-solutions/so-fleet                           2.3.52    b502dec7c098   3 months ago   83.1MB
tddv-nids-soc:5000/security-onion-solutions/so-fleet                2.3.52    b502dec7c098   3 months ago   83.1MB
ghcr.io/security-onion-solutions/so-telegraf                        2.3.52    b14d7d63ecdb   3 months ago   91.5MB
tddv-nids-soc:5000/security-onion-solutions/so-telegraf             2.3.52    b14d7d63ecdb   3 months ago   91.5MB
ghcr.io/security-onion-solutions/so-filebeat                        2.3.52    f8b9004c4381   3 months ago   589MB
tddv-nids-soc:5000/security-onion-solutions/so-filebeat             2.3.52    f8b9004c4381   3 months ago   589MB
ghcr.io/security-onion-solutions/so-elastalert                      2.3.52    1ee772ad1443   3 months ago   434MB
tddv-nids-soc:5000/security-onion-solutions/so-elastalert           2.3.52    1ee772ad1443   3 months ago   434MB
ghcr.io/security-onion-solutions/so-elasticsearch                   2.3.52    c372b807cec9   3 months ago   1.37GB
tddv-nids-soc:5000/security-onion-solutions/so-elasticsearch        2.3.52    c372b807cec9   3 months ago   1.37GB
ghcr.io/security-onion-solutions/so-curator                         2.3.52    74fefeffd5a1   3 months ago   123MB
tddv-nids-soc:5000/security-onion-solutions/so-curator              2.3.52    74fefeffd5a1   3 months ago   123MB
ghcr.io/security-onion-solutions/so-nginx                           2.3.52    acc2602d0aaa   3 months ago   87.4MB
tddv-nids-soc:5000/security-onion-solutions/so-nginx                2.3.52    acc2602d0aaa   3 months ago   87.4MB
ghcr.io/security-onion-solutions/so-acng                            2.3.52    0600c712aa66   3 months ago   82MB
tddv-nids-soc:5000/security-onion-solutions/so-acng                 2.3.52    0600c712aa66   3 months ago   82MB
ghcr.io/security-onion-solutions/so-wazuh                           2.3.52    6371c3a4b20e   3 months ago   882MB
tddv-nids-soc:5000/security-onion-solutions/so-wazuh                2.3.52    6371c3a4b20e   3 months ago   882MB
ghcr.io/security-onion-solutions/so-kibana                          2.3.52    1f6e05b8aaac   3 months ago   1.67GB
tddv-nids-soc:5000/security-onion-solutions/so-kibana               2.3.52    1f6e05b8aaac   3 months ago   1.67GB
ghcr.io/security-onion-solutions/so-suricata                        2.3.52    fb1bd967ec14   3 months ago   735MB
tddv-nids-soc:5000/security-onion-solutions/so-suricata             2.3.52    fb1bd967ec14   3 months ago   735MB
ghcr.io/security-onion-solutions/so-playbook                        2.3.52    794651f5c19e   3 months ago   630MB
tddv-nids-soc:5000/security-onion-solutions/so-playbook             2.3.52    794651f5c19e   3 months ago   630MB
ghcr.io/security-onion-solutions/so-thehive-es                      2.3.52    55f72b2d199a   3 months ago   1.37GB
tddv-nids-soc:5000/security-onion-solutions/so-thehive-es           2.3.52    55f72b2d199a   3 months ago   1.37GB
ghcr.io/security-onion-solutions/registry                           latest    5c4008a25e05   7 months ago   26.2MB**

**Reply by dougburks** [reply]
It's normal for `soup` to keep the previous version's Docker images (in this case 2.3.52).

**Comment by dougburks** [comment, marked as answer]
What is the output of `sudo salt-call state.highstate`?

You could also try a fresh installation of 2.3.70 on a separate machine (perhaps just a VM) and then compare it to your existing installation.

**Reply by JeremyBBD** [reply]
Hi, we tried deploying another VM so that we could check whether we could find a difference between the configuration files but couldn't find any.
We ended up redeploying the servers and thus, cannot troubleshoot the issue any longer.
Thank you for your time.
